collections/corine-land-cover-accounting-layers.yaml

```diff
@@ -98,5 +100,15 @@ CubeDimensions:
     values:
       - CLC_ACC
       - dataMask
+Summaries:
+  raster:bands:
+    - name: CLC_ACC
+      openeo:gsd:
```

Comment:
Please also add everywhere a description of the band. The info should always (in most cases) be somewhere in the yaml file (for the CLC_ACC band it's in line 27 so `description: "Corine Land Cover Accounting Layer"`.

Reply:
Added descriptions of bands